### ISSUE TITLE
NextCloud aborts

### ISSUE THREAD

**Issue by pcza** [nextcloud/android#7578]
### Steps to reproduce
1. ************ CAUSE OF ERROR ************

android.app.RemoteServiceException: Context.startForegroundService() did not then call Service.startForeground(): ServiceRecord{31742d1 u0 com.nextcloud.client/.media.PlayerService}
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2313)
	at android.os.Handler.dispatchMessage(Handler.java:107)
	at android.os.Looper.loop(Looper.java:213)
	at android.app.ActivityThread.main(ActivityThread.java:8178)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:513)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1101)

************ APP INFORMATION ************
ID: com.nextcloud.client
Version: 30140090
Build flavor: gplay

************ DEVICE INFORMATION ************
Brand: HUAWEI
Device: HWBLA
Model: BLA-L29
Id: HUAWEIBLA-L29S
Product: BLA-L29

************ FIRMWARE ************
SDK: 29
Release: 10
Incremental: 10.0.0.159C185

2. 
3. 

### Expected behaviour
- Tell us what should happen

### Actual behaviour
- Tell us what happens

### Can you reproduce this problem on https://try.nextcloud.com?
- Please create a test demo account and see if this still happens there.
- If yes, please open up a bug report
- If not, please verify server setup and ask for help on forum

### Environment data
Android version:

Device model: 

Stock or customized system:

Nextcloud app version:

Nextcloud server version:

Reverse proxy:

#### Web server error log
```
Insert your webserver log here
```

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```
**NOTE:** Be super sure to remove sensitive data like passwords, note that everybody can look here! You can use the Issue Template application to prefill some of the required information: https://apps.nextcloud.com/apps/issuetemplate

**Comment by kosmoz** [comment]
I am getting the same error. To reproduce:

1. Delete any file
2. Wait a few seconds
3. Crash

Device info: Xiaomi Mi 5s running LineageOS 16
App info:  Version 3.14.0 installed from Google Play

**Comment by tobiasKaminsky** [comment]
This will be fixed via 3.14.1 coming today.
